### PR TITLE
Created hargrave-writing-sample

### PR DESCRIPTION
**Pull request by paigehargrave**
This is my 'Writing Exercise' topic for the 'Skill Assessment'. Please note that the 'Bus' and 'Personal vehicle' sections are placeholders, since the assignment was to describe a single form of transportation to Salesforce Tower. Please contact Valkyrie Hunter if whoever first sees this PR has no idea what to do with it! :) Thank you!

### CODE REVIEW

File: modules/getting-started/pages/hargrave-writing-sample.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+endif::[]
+:noindex:
+
+In San Francisco, you have several transportation options for getting to Salesforce Tower:
+
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#rideshare-service[Rideshare service]
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#bus[Bus]  
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#personal-vehicle[Personal vehicle] (car, motorcycle, bicycle)
+
+== Rideshare service
```
Comment (Mulesoft-Mysti) [inline modules/getting-started/pages/hargrave-writing-sample.adoc:13]: Why switch to sentence case for 2nd level heads? (just curious!)

File: modules/getting-started/pages/hargrave-writing-sample.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Rideshare service is the most convenient transportation option if you have a smartphone and cost is not an issue.
+
+To get to Salesforce Tower using a ride share service:
```
Comment (Mulesoft-Mysti) [inline modules/getting-started/pages/hargrave-writing-sample.adoc:17]: rideshare or ride share or ride-share? Should be consistent?

**Comment by Mulesoft-Mysti** [comment]
thank you for taking the initiative to do this sample in asciidoc and in our repo. I'm closing it now so it won't be merged with our product content. Thank you!!!!

File: modules/getting-started/pages/hargrave-writing-sample.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/getting-started/pages/hargrave-writing-sample.adoc:17]: Good catch - thank you! I will update this to be 'rideshare'.

Comment (paigehargrave) [reply modules/getting-started/pages/hargrave-writing-sample.adoc:17]: Was going to fix this to be consistent ('rideshare'), but it looks like editing is disabled.

File: modules/getting-started/pages/hargrave-writing-sample.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/getting-started/pages/hargrave-writing-sample.adoc:13]: @Mulesoft-Mysti Hi - When I googled Salesforce Tower, it seemed to be capitalized that way. My intention was to use sentence case for all headings. Thanks for the quick feedback!

**Pull request closed without merging**